core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1076,6 +1076,15 @@ private void printTableScanInfo(NodeRepresentation nodeOutput, TableScanNode nod
                             });
                 }
             }
+
+            node.getStatistics().ifPresent(statistics -> {
```

Comment:
I know we discussed that but now I have mixed feelings if we should print stats here.
It is quite a lot of output which may clutter the plan. Also, the stats will be present in TS node only in some very specific conditions (after pushdown, if feature is enabled), making it hard to understand why sometimes those are printed and sometimes not.

Maybe instead printing out stats specifically for TS node, if `tableStats` field happens to be filled in, we make it possible for user to request that stats are printed for each node in the plan output. And then TS calculator would be used to compute those.

Reply:
> And then TS calculator would be used to compute those.

This is always done, but without detail (rows, cost).

Note that there is coupling between EXPLAIN output and .... EXPLAIN output used in rule tests. 
If i scratch it from here, it would be an invisible details which is harder to assert on.
But I guess we already have cases like that (eg when rules assert on enforced contraint).

Will drop